adrs/2131725034/README.md

```diff
@@ -0,0 +1,37 @@
+# React Component and Hooks Props Naming
+
+* **State:** Reviewing
+* **Created:** 2022-01-24
+* **Tags:** react
+
+## Context
+
+When it comes to React, they are two categories of properties:
+
+* Data (Information)
+* Callbacks (Events, Control Flow)
+
+Data properties are primarily used for displaying information or transforming
+into another set of data.
+
+While Callbacks are used to control the flow of data such as filtering, sorting,
+for example: `onFilterAccount`, `onSortBy`, etc. Or reacting to events, the
+browser already tries to make distinction between these two categories by
+prefixing the events handlers with `on`, for example: `onClick`, `onChange`,
+etc.
```

Comment:
Might rewrite this as:

_Callbacks are used to control the flow of data. Functions such as `onFilterAccount` and `onSortBy` would be used to filter and sort data.  Callbacks can also react to events. The browser already makes a distinction between data and callback properties by prefixing the event handlers with `on`, for example: `onClick`, `onChange`, etc._

Reply:
Fixed it, hopefully.